apiv2/schema/schema.yaml

```diff
@@ -1723,6 +1696,14 @@ classes:
         multivalued: true
         inverse: Frame.run
         range: Frame
+      gain_files:
```

Comment:
Could we also update the description of the `run.dataset` in [here](https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/280/files#diff-145ab66c6e4310481f4a1eaf8c311191a0349f39f7d7d9ac92207ecdfda3a6c4R1688)?